syntax = "proto3";

package oracle;

service Oracle {
    rpc ListGraphs(ListGraphsRequest) returns (ListGraphsResponse) {}
}

message ListGraphsRequest {
    //Domain domain = 1;
    //Predicate predicate = 2;
    string page_token = 3;
    uint64 max_page_size = 4;
}

message ListGraphsResponse {
    //repeated Graph graphs = 1;
    string next_page_token = 2;
}
